hack/make.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+set -e
+
+# This script builds various binary artifacts from a checkout of the docker
+# source code.
+#
+# Requirements:
+# - The current directory should be a checkout of the docker source code
```

Comment:
Maybe we should update that :stuck_out_tongue: (and a little up, and a little down too)